helm/cloud-provider-cloud-director-app/charts/cloud-director-named-disk-csi-driver/values.yaml

```diff
@@ -0,0 +1,42 @@
+configmap:
+  site: "" # VCD endpoint with the format https://VCD_HOST. No trailing '/'
+  org: "" # VCD organization name where the cluster should be deployed
+  ovdc: "" # VCD virtual datacenter name where the cluster should be deployed
+  vAppName: "" # "name" field of VCDCluster object in management cluster
+  clusterid: "" # "Infra Id" field of VCDCluster object in management cluster
+
+secret:
+  create: true # Set to false if you create the secret somewhere else
+  username: ""
+  password: ""
+  refreshToken: ""
+
+statefulset:

```

Comment:
Don't need to add statefulset or daemonset as a top level here. k8s resources names are not used here. Something that is more meaningful / more human readable should be used.  IE.  csi-controller

Reply:
Changed to `controllerStatefulset`